cmd/client.go

```diff
@@ -138,11 +138,15 @@ func newCliCommand() *cobra.Command {
 				return errors.Annotate(err, "fail to validate TLS settings")
 			}
 			if tlsConfig != nil {
-				if strings.Contains(cliPdAddr, "http://") {
+				if strings.HasPrefix(cliPdAddr, HTTP) {
```

Comment:
Why not parse the URL to check the scheme information?

Reply:
Yes, it is best to do a URL validate first: https://github.com/pingcap/ticdc/pull/2138#issuecomment-867303878

I will add that step.

Reply:
Added. Could you please take again?